pkg/logging/logging.go

```diff
@@ -133,6 +137,33 @@ func LoadLogConfig(dataStore, ns, id string) (LogConfig, error) {
 	return logConfig, nil
 }
 
+func loggingProcessAdapter(driver Driver, config *logging.Config) error {
+	var wg sync.WaitGroup
+	wg.Add(3)
+	stdout := make(chan string, 10000)
+	stderr := make(chan string, 10000)
+	processLogFunc := func(reader io.Reader, dataChan chan string) {
+		defer wg.Done()
+		defer close(dataChan)
```

Comment:
`processLogFunc` is called twice downside, should make sure to close the channel on the last call.